float xPos
float yPos
float zPos

float chassisPitch
float chassisYaw
float chassisRoll

float turretPitch
float turretYaw